app/views/container_service/_main.html.haml

```diff
@@ -1,8 +1,15 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
-    = render :partial => "shared/summary/textual_multilabel", :locals => { :title => _("Port Configurations"), :items => textual_group_port_configs }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
```

Comment:
@dkorn I don't mind you opening a *separate* PR to fix cosmetics that can be reviewed independently of a bug fix or enhancement PR.